config-model/src/main/java/com/yahoo/vespa/model/container/ApplicationContainer.java

```diff
@@ -64,10 +66,23 @@ protected ContainerServiceType myServiceType() {
     /** Returns the jvm arguments this should start with */
     @Override
     public String getJvmOptions() {
+        StringBuilder b = new StringBuilder();
+        if (isHostedVespa) {
+            if (hasDocproc()) {
+                b.append(ApplicationContainer.defaultHostedJVMArgs).append(' ');
+            }
+            if (enableServerOcspStapling) {
+                b.append("-Djdk.tls.server.enableStatusRequestExtension=true ")
+                        .append("-Djdk.tls.stapling.responseTimeout=2000 ")
+                        .append("-Djdk.tls.stapling.cacheSize=256 ")
+                        .append("-Djdk.tls.stapling.cacheLifetime=3600 ");
+            }
+        }
         String jvmArgs = super.getJvmOptions();
-        return isHostedVespa && hasDocproc()
-                ? ("".equals(jvmArgs) ? defaultHostedJVMArgs : defaultHostedJVMArgs + " " + jvmArgs)
-                : jvmArgs;
+        if (!jvmArgs.isBlank()) {
+             b.append(jvmArgs.trim()).append(' ');
```

Comment:
Consider removing `append(' ')` as this is always removed by line 85 `return b.toString().trim()`